src/main/java/meteordevelopment/meteorclient/systems/modules/player/AutoEat.java

```diff
@@ -77,6 +77,13 @@
         .build()
     );
 
+    private final Setting<Boolean> avoidOvereating = sgHunger.add(new BoolSetting.Builder()
+        .name("avoid-overeating")
+        .description("Avoid eating foods that would bring your hunger over full.")
```

Comment:
I kinda dislike the naming, tbh I don't have a better replacement tho.
What about something like "that would make your saturation overshoot the hunger bar"?

Reply:
Not saturation though, you sure?

Reply:
Maybe replace it with  "hunger level" then?